packages/react-server-dom-fb/src/ReactFlightReferencesFB.js

```diff
@@ -7,15 +7,16 @@
  * @flow
  */
 
-export opaque type ClientManifest = mixed;
+export type ClientManifest = null;
 
 // eslint-disable-next-line no-unused-vars
 export type ServerReference<T> = string;
 
 // eslint-disable-next-line no-unused-vars
```

Comment:
are the `// eslint-disable-next-line no-unused-vars` actually needed?

Reply:
This is just to keep the same interface across different configurations, these types are using generic T, which is unused in this specific case.